binder/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM trungleduc/jupytercad-env:0.1.1
```

Comment:
Should we move this docker image under the jupytercad repo?

Reply:
You mean the docker file or creating `jupytercad` account on the docker hub?

Reply:
I mean the Dockerfile that was used for publishing `trungleduc/jupytercad-env` on Dockerhub?

Reply:
ah ok, where should I put it?

Reply:
Maybe under a `docker` directory on this repo?

Reply:
For the account on docker hub, we could probably use the https://hub.docker.com/u/quantstack one

Reply:
Do you have the right to push images to this account?